docs/pom.xml

```diff
@@ -40,11 +40,96 @@
+                                    <arg value="exec" />
+                                    <arg value="jekyll-war-build" />
+                                    <arg value="sh" />
+                                    <arg value="-c" />
+                                    <arg value="mkdir -p /srv/jekyll/assets/imgs/ ; cp -rf /srv/jekyll/target/che-repo/jekyll/docs/assets/imgs/* /srv/jekyll/assets/imgs/" />
+                                </exec>
+                                <exec dir="${project.build.directory}" executable="docker" failonerror="true">
+                                    <arg value="exec" />
+                                    <arg value="jekyll-war-build" />
+                                    <arg value="sh" />
+                                    <arg value="-c" />
+                                    <arg value="rm -r /srv/jekyll/target" />
+                                </exec>
+                                <exec dir="${project.build.directory}" executable="docker" failonerror="true">
+                                    <arg value="exec" />
+                                    <arg value="jekyll-war-build" />
+                                    <arg value="sh" />
+                                    <arg value="-c" />
+                                    <arg value="chown -R jekyll:jekyll /srv/jekyll" />
+                                </exec>
+                                <exec dir="${project.build.directory}" executable="docker" failonerror="true">
+                                    <arg value="exec" />
+                                    <arg value="jekyll-war-build" />
+                                    <arg value="jekyll" />
                                     <arg value="build" />
-                                    <arg value="--config" />
-                                    <arg value="assembly/build_war.yml" />
                                 </exec>
+                                <exec dir="${project.build.directory}" executable="docker" failonerror="true">
+                                    <arg value="cp" />
+                                    <arg value="jekyll-war-build:/srv/jekyll/_site" />
+                                    <arg value="${project.build.directory}" />
+                                </exec>
+                                <exec dir="${project.basedir}" executable="docker" failonerror="true">
+                                    <arg value="rm" />
+                                    <arg value="-f" />
+                                    <arg value="jekyll-war-build" />
+                                </exec>
+                                <delete file="${project.build.directory}/_site/feed.xml" />
+                                <delete file="${project.build.directory}/_site/feed.xslt.xml" />
+                                <delete file="${project.build.directory}/_site/sitemap.xml" />
```

Comment:
Instead of launching 15 times the docker container, I think we should give a shell script to docker container and only run once
Also docker run command should be using --rm flag no ? as it will end with several non-cleaned containers